scripts/lib/mkcloud-libvirt.sh

```diff
@@ -64,14 +64,16 @@ function libvirt_net_start()
 
 iptables -t nat -F PREROUTING
 for i in 22 80 443 3000 4000 4040 5000; do
-    iptables -I FORWARD -p tcp --dport \$i -j ACCEPT
     for host in 10 $nodehostips ; do
         offset=80
         [ "\$host" = 10 ] && offset=10
         iptables -t nat -I PREROUTING -p tcp --dport \$((\$i + \$host - \$offset + 1100)) -j DNAT --to-destination $net_admin.\$host:\$i
     done
 done
 iptables -t nat -I PREROUTING -p tcp --dport 6080 -j DNAT --to-destination $net_public.2
+for x in D I ; do
+    iptables -$x FORWARD -d $net_public.0/24 -j ACCEPT
```

Comment:
My test showed that this is broken - it should be `-\$x` because of the `<<EOS`.